pytorch_lightning/utilities/device_dtype_mixin.py

```diff
@@ -21,17 +21,17 @@
 class DeviceDtypeModuleMixin(Module):
     __jit_unused_properties__ = ['device', 'dtype']
 
-    def __init__(self):
+    def __init__(self) -> None:
         super().__init__()
-        self._dtype = torch.get_default_dtype()
+        self._dtype: Union[str, torch.dtype] = torch.get_default_dtype()
         self._device = torch.device('cpu')
 
     @property
     def dtype(self) -> Union[str, torch.dtype]:
         return self._dtype
 
     @dtype.setter
-    def dtype(self, new_dtype: Union[str, torch.dtype]):
+    def dtype(self, new_dtype: Union[str, torch.dtype]) -> RuntimeError:
```

Comment:
Is this return allowed? Can you share some resource about it?

Reply:
Oh sorry, exceptions shouldn't be included in return types. I put just `None` there.